native/src/ElapsedTimer.h

```diff
@@ -7,10 +7,10 @@ using namespace std::chrono;
 namespace OsmAnd {
 class ElapsedTimer {
    private:
-	high_resolution_clock::duration elapsed;
-	high_resolution_clock::time_point startPoint;
 	bool isEnabled;
 	bool isRunning;
+	high_resolution_clock::duration elapsed;
+	high_resolution_clock::time_point startPoint;
```

Comment:
Why change position. Do you want change ownership of this rows?

Reply:
Position changed because C++ constructor default values initializator seqence should be the same as values declated in the class. Warning fixed.